test/conformance/ptp/ptp.go

```diff
@@ -943,3 +989,124 @@ func ptpEventEnabled() bool {
 	}
 	return ptpConfig.Spec.EventConfig.EnableEventPublisher
 }
+
+func checkAndGetPtpRunningPods(fullConfig testconfig.TestConfig) []v1core.Pod {
+	ptpPods, err := client.Client.CoreV1().Pods(PtpLinuxDaemonNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: "app=linuxptp-daemon"})
+	Expect(err).NotTo(HaveOccurred())
+	Expect(len(ptpPods.Items)).To(BeNumerically(">", 0), "linuxptp-daemon is not deployed on cluster")
```

Comment:
Shouldn't we have a time wait loop here to wait after nodes have been pinging successfully and before the oc command is available

Reply:
This is done in the function `rebootNode` in the code 

```java
// Wait for all nodes to be reachable now after the restart
	const reachable = true
	for _, node := range nodes.Items {
		testnode.WaitForNodeReachability(&node, 10*time.Minute, reachable)
	}`